### PR TITLE
Fixed Style and Added Countdown on Resend Mail.

### PR DESCRIPTION
**Pull request by Todua992**
## Description

After the User Testing and new PBI of background change on Reset password Modal. I made some improvements. The Password Reset now has the same style as when doing Email Verification. 

**PBI**
#107 

On top of that, I made some changes on Email Verification after User Test. 
Features : 
Added Cooldown on Resend Verification Code 
When something wrong, The user gets notification of Error message.
Bug Fixes : 
The user cant spam resend verification button anymore. 
The user was not Receiving any errors when something went wrong.
## Related Issues

!!!!!!!!!!!This branch works only with the BackEnd Branch REGISTRATION-DEADLOCK!!!!!!
## Checklist

- [x] Code has been tested locally and passes all relevant tests.
- [x] Documentation has been updated to reflect the changes, if applicable.
- [x] Code follows the established coding style and guidelines of the project.
- [x] All new and existing tests related to the changes have passed.
- [x] Any necessary dependencies or new packages have been properly documented.
- [x] Pull request title and description are clear and descriptive.
- [x] Reviewers have been assigned to the pull request.
- [x] Any potential security implications have been considered and addressed.
- [x] Performance impact of the changes has been evaluated, if relevant.
- [x] Changes have been approved by Product Owner
- [x] Code has been analyzed with CodeScene
- [x] I have reviewed the pull request myself

## Screenshots 
**Registration** 

image

image

image

**Reset Password** 
image

## If mobile/frontend pull request, what version of the backend is it stable, and running on? 
!!!!!!!!!!!This branch works only with the BackEnd Branch REGISTRATION-DEADLOCK!!!!!!

### CODE REVIEW

**Review by LazarShockX — APPROVED** [review]
You have my approval 🩵

File: app/src/components/emailVerification/NavigationFooter.tsx [hunk 1]
```diff
@@ -60,75 +72,61 @@ export default function NavigationFooter(props: propsType): JSX.Element {
...
-          className="py-2 px-7 mt-8 bg-primary hover:bg-gray-100 border border-primary hover:text-primary text-white w-full transition ease-in duration-200 text-center text-base font-semibold shadow-md focus:ring-offset-2  rounded"
-        >
-          {!props.codeVerified ? 'Continuar' : 'Redefinir senha'} {/* Continue or reset password */}
-        </button>
-      </label>
-  
-      <label className="flex justify-center w-full mt-4">
-  <div className="flex items-center gap-2">
-    <p>O código não chegou?</p>
+    <div className="">
```
Comment (MartinJenzen) [inline app/src/components/emailVerification/NavigationFooter.tsx:103]: A bit weird with an empty string in this className ;)

File: app/src/components/passwordRecovery/PasswordRecoveryModal.tsx [hunk 2]
```diff
@@ -157,38 +157,43 @@ const PasswordRecoveryModal = (props: propTypes) : JSX.Element => {
...
-              setPasswordContainsLetter={setPasswordContainsLetter}
-              setPasswordLengthValid={setPasswordLengthValid}
-            />
-          }
-          <NavigationFooter codeVerified={codeVerified} />
-        </div>
-      </HandleContinueContext.Provider>
+    <div id="password-reset-modal" className="absolute inset-0 grid place-items-center">
+  {/* Background overlay */}
+  <div className="absolute inset-0 bg-[#383838] opacity-60"></div>
```
Comment (MartinJenzen) [inline app/src/components/passwordRecovery/PasswordRecoveryModal.tsx:162]: The indentation from here on line 162 (incl. 161) and down has a bit of retardation 🤡😉

**Review by MartinJenzen — APPROVED** [review]
A few _small_ errors, but otherwise: nice job 👍

File: app/src/components/emailVerification/NavigationFooter.tsx (hunk 1 above)
Comment (Todua992) [reply app/src/components/emailVerification/NavigationFooter.tsx:103]: Very Sharp eyes XD

**Pull request merged**